benchmarks/bench_rmsnorm_vs_empirical_sol.py

```
@@ -0,0 +1,841 @@
+    for batch_size in args.batch_sizes:
+        for hidden_size in args.hidden_sizes:
+            current += 1
+            try:
+                row = run_case(
+                    args=args,
+                    batch_size=batch_size,
+                    hidden_size=hidden_size,
+                    input_dtype=input_dtype,
+                    esol_args=esol_args,
+                )
+            except Exception as exc:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)